cumulusci.yml

```diff
@@ -198,6 +232,18 @@ flows:
                     namespaced_org: True
                     skip_record_types: True
 
+    config_platformencryption:
```

Comment:
@jofsky @lifewithryan This caught my eye - just wanted to see if any of this was duplicate work of what I already implemented in #955? Some of it looks the same, some of it potentially new. There's a `setup_encryption` flow and `deploy_encryption_permset`, `assign_encryption_permset`, & `create_tenant_secret` tasks.

Reply:
Yea that's probably duplicate, Josh's items also then configures the encrypted fields for us as well so we don't have to touch anything manually. So we will want to either leverage your tasks or remove yours and just leverage Josh's. I would suggest just leverage Josh's now since he's got pretty much a soup to nuts implementation.
I apologize for not being clearer that we already had automation to setup the key, etc.

Reply:
I'd suggest we use the existing tasks/flow, and append `platformencryption_deploy_field_configuration` to `setup_encryption`.

Reply:
I see it also already documented in https://github.com/SalesforceFoundation/EDA/blob/master/documentation/automation.md -- we could consider expanding on that documentation a bit, perhaps.

There's also a Quip doc I spun up a few months ago: https://salesforce.quip.com/hd1UAWXDUqJB